client/src/pages/OneCampaign/index.jsx

```diff
@@ -0,0 +1,126 @@
+import {
+  Col,
+  Divider,
+  Image,
+  message,
+  Progress,
+  Row,
+  Statistic,
+  Typography,
+} from 'antd';
+import axios from 'axios';
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import './style.less';
+import { TeamOutlined } from '@ant-design/icons';
+import { DonationButton } from '../../components';
+
+const { Title, Paragraph } = Typography;
+function OneCampaign({ id }) {
```

Comment:
Call it just Campaign, and you don't need to pass the id here as you can access it through the params